app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def update_posts_likes_counter
+    post.increment!(:likes_counter)
+  end
+end
```

Comment:
It will be nice to use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_posts_likes_counter method every time a new post has been saved to the database, then make the update_posts_likes_counter method private.
```ruby
# app/models/like.rb
  after_save :update_posts_likes_counter

  private

  def update_posts_likes_counter
    author.increment!(:posts_likes_counter)
  end
```